experiment.py

```diff
@@ -70,6 +70,8 @@
 parser.add_argument('--lambda_kd', type=float, default=1.0, help='trade-off parameter for kd loss')
 parser.add_argument('--kd_epochs_first', type=int, default=200, help='use kd for the first several epochs')
 parser.add_argument('--kd_epochs_every', type=int, default=1, help='use kd every x epochs')
```

Comment:
How about removing "default=False" since you already use 'store_true'? eg. for `pretrained` and `evaluate`, they are 
```
parser.add_argument('-e', '--evaluate', dest='evaluate', action='store_true',
                    help='evaluate model on validation set')
parser.add_argument('--pretrained', dest='pretrained', action='store_true',
                    help='use pre-trained model')
```

Reply:
updated